src/main/java/seedu/address/Tasks/Deadline.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Deadline class represents a deadline task for the ProfBook.
+ * It extends the Task class and provides additional methods specific to deadline tasks.
+ */
+public class Deadline extends Task {
+    String FormattedTime;
+    LocalDateTime DueBy;
+    String unformattedTime;
+
+    /**
+     * Constructs a new Deadline object with the given description and deadline.
+     * @param description the description of the Deadline task
+     * @param deadline the deadline of the Deadline task in the format "dd/MM/yyyy HHmm"
+     */
+    public Deadline(String description, String deadline) {
+        super(description);
+        this.unformattedTime = deadline;
+        String[] parts = deadline.split("[/ ]");
+        int day = Integer.parseInt(parts[0]);
+        int month = Integer.parseInt(parts[1]);
+        int year = Integer.parseInt(parts[2]);
+        int hour = Integer.parseInt(parts[3].substring(0, 2));
+        int minute = Integer.parseInt(parts[3].substring(2));
+        LocalDateTime dateTime = LocalDateTime.of(year, month, day, hour, minute);
```

Comment:
For deadline field, is ok to directly pass in a LocatDate object to the constructor as the parsing logic should be covered by parser